sys/linux/fuse_arm64.const

```diff
@@ -18,7 +18,7 @@ FUSE_FLOCK_LOCKS = 1024
 FUSE_HANDLE_KILLPRIV = 524288
 FUSE_HAS_IOCTL_DIR = 2048
 FUSE_IOCTL_RETRY = 4
-FUSE_KERNEL_MINOR_VERSION = 27
+FUSE_KERNEL_MINOR_VERSION = 28
```

Comment:
We already have this:
syzkaller$ grep FUSE_KERNEL_MINOR_VERSION sys/linux/*.const
sys/linux/fuse_386.const:FUSE_KERNEL_MINOR_VERSION = 28
sys/linux/fuse_amd64.const:FUSE_KERNEL_MINOR_VERSION = 28
sys/linux/fuse_arm64.const:FUSE_KERNEL_MINOR_VERSION = 28
sys/linux/fuse_arm.const:FUSE_KERNEL_MINOR_VERSION = 28
sys/linux/fuse_ppc64le.const:FUSE_KERNEL_MINOR_VERSION = 28